doc/classes/WorkerThreadPool.xml

```diff
@@ -1,8 +1,69 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="WorkerThreadPool" inherits="Object" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A singleton that executes [Callable]s in a multithreaded way.
 	</brief_description>
 	<description>
+		Just like the signal system, the [WorkerThreadPool] uses [Callable]s. The [WorkerThreadPool] executes them in a multithreaded way. You can schedule the worker to process one [Callable] as an individual task. Which is then executed in another thread. Or schedule multiple elements of one [Callable] in a group task. These are executed in one or more threads. You decide. Set a task as [b]high_priority[/b] to let the worker process them earlier than others.
+		To schedule [Callable]s, pass them on to [method add_task] and [method add_group_task]:
+		[codeblocks]
+		[gdscript]
+		func do_something() -&gt; void:
+		    var is_very_important : bool = true
+		    var task_id : int = WorkerThreadPool.add_task(self.load_cheese_cake, is_very_important, "do it fast, but elsewhere")
+		    # and now we wait…
+		    WorkerThreadPool.wait_for_task_completion(task_id)
+
+		    # Now together
+		    # But only if this class inherits from [Object].
```

Comment:
What does this mean?

"Together" with what?

Why only if a class inherits from `Object`?

It's not very clear.

Reply:
> "Together" with what?

changed to `Or schedule the Callable processing multiple times.`

> Why only if a class inherits from Object?

Since I wrote the docs, the Callable docs have been improved: Removed the `Object` inheritance info